### PR TITLE
MTC-635: extended related orders info

### PR DESCRIPTION
**Pull request by andrey-anokhin**

### CODE REVIEW

File: src/MarginTrading.Backend/Controllers/OrdersController.cs [hunk 1]
```diff
@@ -302,5 +302,20 @@ private OriginatorType GetOriginator(OriginatorTypeContract? originator)
 
             return originator.ToType<OriginatorType>();
         }
+
+        private List<Order> GetRalatedOrders(Order order)
```
Comment (GSailen) [inline src/MarginTrading.Backend/Controllers/OrdersController.cs]: 1. typo
2. may be reused in RabbitMqNotifyService.cs if it is an extension or placed to some common service.

File: src/MarginTrading.Backend/Controllers/PositionsController.cs [hunk 2]
```diff
@@ -225,8 +226,25 @@ public void StartSpecialLiquidation(string[] positionIds, [CanBeNull] string acc
             });
         }
 
-        internal static OpenPositionContract Convert(Position position)
+        private OpenPositionContract Convert(Position position)
         {
+            var relatedOrders = new List<RelatedOrderInfoContract>();
```
Comment (GSailen) [inline src/MarginTrading.Backend/Controllers/PositionsController.cs]: Same again, reuse GetRalatedOrders

Comment (andrey-anokhin) [reply src/MarginTrading.Backend/Controllers/PositionsController.cs]: I think it's not worth it. Agreed to leave as is.

File: src/MarginTrading.Backend/Controllers/OrdersController.cs (hunk 1 above)
Comment (andrey-anokhin) [reply src/MarginTrading.Backend/Controllers/OrdersController.cs]: Fixed typo

**Pull request merged**